src/HLevel.ard

```diff
@@ -99,9 +122,9 @@
   | suc n => \lam a a' => HLevel-retracts (pmapSection s) (p (s.f a) (s.f a'))
 
 -- | hLevels are closed under pi-types
-\func HLevels-pi {A : \Type} (B : A -> \Type) {n : Nat} (p : \Pi (a : A) -> B a ofHLevel_-1+ n) : (\Pi (a : A) -> B a) ofHLevel_-1+ n \elim n
+\func HLevels-pi {A : \Type} (B : A -> \Type) (n : Nat) (p : \Pi (a : A) -> B a ofHLevel_-1+ n) : (\Pi (a : A) -> B a) ofHLevel_-1+ n \elim n
   | 0 => \lam f f' => path (\lam i a => p a (f a) (f' a) @ i)
-  | suc n => \lam f f' => HLevel-retracts (piEquiv B f f') (HLevels-pi (\lam a => f a = f' a) (\lam a => p a (f a) (f' a)))
+  | suc n => \lam f f' => HLevel-retracts (piEquiv B f f') (HLevels-pi (\lam a => f a = f' a) n (\lam a => p a (f a) (f' a)))
```

Comment:
Made this explicit because all of its usages except its recursion need to explicit specify `n`

Reply:
That's not true. If `n` is not a constructor, it can be inferred.

Reply:
There are times the return type is needed to infer other things, but you need an value of n to let Arend reduce the return type of this function. Anyway, look at the two usages (one existing and one newly added in the pr), you'll see that they cannot be inferred.

Reply:
It can be inferred either from the type of `p` or from the result type. I see 3 usages in `Homotopy.Truncation` where it is inferred. There is only one usage in `Equiv.Fiber` where it is also constant, so it is not inferred.

Reply:
See the other comment, I `@` you

Reply:
Fixed